tests/models/test_torchscript.py

```diff
@@ -139,6 +144,36 @@ def test_torchscript_save_load(tmpdir, modelclass):
     assert torch.allclose(next(script.parameters()), next(loaded_script.parameters()))
 
 
+_DUMMY_PRFEIX = "dummy"
+_PREFIX_SEPARATOR = "://"
```

Comment:
Are these set as variables (instead of hardcoded) for a reason? Do you expect that other tests would need to modify them?

Reply:
i think a dummy custom filesystem could be helpful to ensure fsspec works as expected, but this is not the right spot to land this. for now, i will move it into the test. i initially avoided putting this into the test to avoid registering the filesysem 3 times